attestation-service/verifier/src/lib.rs

```diff
@@ -152,6 +166,13 @@ pub trait Verifier {
         expected_report_data: &ReportData,
         expected_init_data_hash: &InitDataHash,
     ) -> Result<TeeEvidenceParsedClaim>;
+
+    async fn generate_supplemental_challenge(
```

Comment:
we probably want to add api docs for this trait fn

Reply:
Added, thanks!